project/src/components/reviews-form/reviews-form.tsx

```diff
@@ -0,0 +1,78 @@
+import { useState, Fragment } from 'react';
+
+const Limit = {
+  MIN: 50,
+  MAX: 300,
+};
+
+const ratings = [
+  ['5', 'perfect'],
+  ['4', 'good'],
+  ['3', 'not bad'],
+  ['2', 'badly'],
+  ['1', 'terribly'],
+];
+
+function ReviewsForm() {
```

Comment:
Это форма 1 коммента
```suggestion
function ReviewForm() {
```